releases/v1.1.2.toml

```diff
@@ -0,0 +1,24 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.1.1"
+
+pre_release = false
+
+preface = """\
+This is the second patch release for the `containerd` 1.1 release. This
+includes an issue fix for CRI.
+## CRI Plugin
+Fix an issue that container log line longer than 4kb is cut into
```

Comment:
I think this would sentence would read better as either `that a container log line longer than 4kb is` or `that container log lines longer than 4kb are`

Reply:
Done.